File: cpp/25a_1409A_YetAnotherTwoIntegersProblem.cpp
#include<bits/stdc++.h>

int main(){
    int t, a, b, temp;
    std::vector<int> ans;
    std::cin>>t;
    for(int i = 0; i < t; i++){
        std::cin>>a>>b;
        temp = (abs(a - b) / 10);
        if(abs(a - b) % 10 != 0){
            temp ++;
        }
        ans.push_back(temp);
    }
    for(auto i : ans){
        std::cout<<i<<"\n";
    }
    return 0;
}

File: python/25a_1409A_YetAnotherTwoIntegersProblem.py
def main():
    t = int(input())
    ans = []
    for _ in range(t):
        a, b = input().split()
        a, b = int(a), int(b)
        temp = abs(a - b) // 10
        if abs(a - b) % 10 != 0:
            temp += 1
        ans.append(temp)
    for i in ans:
        print(i)

if __name__ == "__main__":
    main()